application/frontend/config/main.php

```diff
@@ -2,25 +2,20 @@
 
 /* Get frontend-specific config settings from ENV vars or set defaults. */
 $frontCookieKey = getenv('FRONT_COOKIE_KEY') ?: null;
+const UID_ROUTE_PATTERN = '<uid:([a-zA-Z0-9_\-]{32})>';
```

Comment:
As things currently stand, this matches the values we're using for UIDs. Is there some way we can make sure we know to update this if we ever change the possible format of a UID?  (e.g. - Should/could we add a unit test that simply checks ten values returned by our UID-generating code against this pattern [retrieved from the same place this copy of it is retrieved from]?).

I guess I'm suggesting that some static methods be added in a single class that handles this regex and the generation of our UIDs, and using those static methods anywhere we need either this regex or a new UID.